rush/rush/src/utilities/InstallManager.ts

```diff
@@ -180,7 +176,7 @@ export default class InstallManager {
       console.log(os.EOL + 'Running "npm install" in ' + npmToolFolder);
 
       // NOTE: Here we use whatever version of NPM we happen to find in the PATH
-      Utilities.executeCommandWithRetry('npm', ['install'], MAX_INSTALL_ATTEMPTS, npmToolFolder);
+      Utilities.executeCommandWithRetry('npm', ['install', '--no-optional'], MAX_INSTALL_ATTEMPTS, npmToolFolder);
```

Comment:
Also, what was the outcome of your shrinkwrap-diffing experiment?

---
In reply to: [115061095](https://github.com/Microsoft/web-build-tools/pull/175#discussion_r115061095) [](ancestors = 115061095)